src/test/java/com/epam/tc/hw1/SumTest.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SumTest extends AbstractTest {
+
+    @Test
+    public void sumTest() {
+        double res = calculator.sum(1, 2);
+        assertThat(res).isEqualTo(3);
+    }
```

Comment:
Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

Reply:
I removed simply tests and expanded the data set in the existing ones through the DataProviders